aws/scylla_create_devices

```diff
@@ -189,6 +188,16 @@ def get_disk_bundles():
         config_array(typemap[t], role[t], mdidx)
         mdidx += 1
 
-
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Disk creation script for Scylla.')
+    parser.add_argument('--raid-device', dest='raid_device', action='store',
+                        help='location of RAID device', default="/dev/md%d")
```

Comment:
"/dev/md%d" ? who is going to use this formatting ?

Reply:
removed this param